File: MiniDOMFinanziaria/Finanziaria/ValutazioneAzienda/CQSPDValutazioneAziendaCursor.cs
using System;

namespace minidom
{
    public partial class Finanziaria
    {
        [Serializable]
        public class CQSPDValutazioneAziendaCursor : Databases.DBObjectCursor<CQSPDValutazioneAzienda>
        {
            private DBCursorField<int> m_IDAzienda = new DBCursorField<int>("IDAzienda");
            private DBCursorStringField m_NomeAzienda = new DBCursorStringField("NomeAzienda");
            private DBCursorField<int> m_IDOperatore = new DBCursorField<int>("IDOperatore");
            private DBCursorStringField m_NomeOperatore = new DBCursorStringField("NomeOperatore");
            private DBCursorStringField m_TipoFonte = new DBCursorStringField("TipoFonte");
            private DBCursorField<int> m_IDFonte = new DBCursorField<int>("IDFonte");
            private DBCursorStringField m_NomeFonte = new DBCursorStringField("NomeFonte");
            private DBCursorField<decimal> m_CapitaleSociale = new DBCursorField<decimal>("CapitaleSociale");
            private DBCursorField<int> m_NumeroDipendenti = new DBCursorField<int>("NumeroDipendenti");
            private DBCursorField<decimal> m_FatturatoAnnuo = new DBCursorField<decimal>("FatturatoAnnuo");
            private DBCursorField<double> m_RapportoTFR_VN = new DBCursorField<double>("RapportoTFR_VN");
            private DBCursorField<int> m_Rating = new DBCursorField<int>("Rating");
            private DBCursorField<DateTime> m_DataRevisione = new DBCursorField<DateTime>("DataRevisione");
            private DBCursorField<DateTime> m_DataScadenzaRevisione = new DBCursorField<DateTime>("DataScadenzaRevisioneione");
            private DBCursorStringField m_StatoAzienda = new DBCursorStringField("StatoAzienda");
            private DBCursorStringField m_DettaglioStatoAzienda = new DBCursorStringField("DettaglioStatoAzienda");
            private DBCursorField<int> m_GiorniAnticipoEstinzione = new DBCursorField<int>("GiorniAnticipoEstinzione");
            private DBCursorField<CQSPDValutazioneAziendaFlags> m_Flags = new DBCursorField<CQSPDValutazioneAziendaFlags>("Flags");

            public CQSPDValutazioneAziendaCursor()
            {
            }

            public DBCursorField<int> IDOperatore
            {
                get
                {
                    return m_IDOperatore;
                }
            }

            public DBCursorStringField NomeOperatore
            {
                get
                {
                    return m_NomeOperatore;
                }
            }

            public DBCursorField<int> IDAzienda
            {
                get
                {
                    return m_IDAzienda;
                }
            }

            public DBCursorStringField NomeAzienda
            {
                get
                {
                    return m_NomeAzienda;
                }
            }

            public DBCursorStringField TipoFonte
            {
                get
                {
                    return m_TipoFonte;
                }
            }

            public DBCursorField<int> IDFonte
            {
                get
                {
                    return m_IDFonte;
                }
            }

            public DBCursorStringField NomeFonte
            {
                get
                {
                    return m_NomeFonte;
                }
            }

            public DBCursorField<decimal> CapitaleSociale
            {
                get
                {
                    return m_CapitaleSociale;
                }
            }

            public DBCursorField<int> NumeroDipendenti
            {
                get
                {
                    return m_NumeroDipendenti;
                }
            }

            public DBCursorField<decimal> FatturatoAnnuo
            {
                get
                {
                    return m_FatturatoAnnuo;
                }
            }

            public DBCursorField<double> RapportoTFR_VN
            {
                get
                {
                    return m_RapportoTFR_VN;
                }
            }

            public DBCursorField<int> Rating
            {
                get
                {
                    return m_Rating;
                }
            }

            public DBCursorField<DateTime> DataRevisione
            {
                get
                {
                    return m_DataRevisione;
                }
            }

            public DBCursorField<DateTime> DataScadenzaRevisione
            {
                get
                {
                    return m_DataScadenzaRevisione;
                }
            }

            public DBCursorStringField StatoAzienda
            {
                get
                {
                    return m_StatoAzienda;
                }
            }

            public DBCursorStringField DettaglioStatoAzienda
            {
                get
                {
                    return m_DettaglioStatoAzienda;
                }
            }

            public DBCursorField<int> GiorniAnticipoEstinzione
            {
                get
                {
                    return m_GiorniAnticipoEstinzione;
                }
            }

            public DBCursorField<CQSPDValutazioneAziendaFlags> Flags
            {
                get
                {
                    return m_Flags;
                }
            }

            protected override Sistema.CModule GetModule()
            {
                return ValutazioniAzienda.Module;
            }

            public override string GetTableName()
            {
                return "tbl_CQSPDValutazioniAzienda";
            }

            protected override Databases.CDBConnection GetConnection()
            {
                return Database;
            }
        }
    }
}

File: MiniDOMFinanziaria/Finanziaria/ValutazioneAzienda/CQSPDValutazioneAziendaCursor.vb
Imports minidom
Imports minidom.Databases
Imports minidom.Sistema
Imports minidom.Anagrafica



Partial Public Class Finanziaria


    <Serializable>
    Public Class CQSPDValutazioneAziendaCursor
        Inherits DBObjectCursor(Of CQSPDValutazioneAzienda)

        Private m_IDAzienda As New CCursorField(Of Integer)("IDAzienda")
        Private m_NomeAzienda As New CCursorFieldObj(Of String)("NomeAzienda")
        Private m_IDOperatore As New CCursorField(Of Integer)("IDOperatore")
        Private m_NomeOperatore As New CCursorFieldObj(Of String)("NomeOperatore")
        Private m_TipoFonte As New CCursorFieldObj(Of String)("TipoFonte")
        Private m_IDFonte As New CCursorField(Of Integer)("IDFonte")
        Private m_NomeFonte As New CCursorFieldObj(Of String)("NomeFonte")
        Private m_CapitaleSociale As New CCursorField(Of Decimal)("CapitaleSociale")
        Private m_NumeroDipendenti As New CCursorField(Of Integer)("NumeroDipendenti")
        Private m_FatturatoAnnuo As New CCursorField(Of Decimal)("FatturatoAnnuo")
        Private m_RapportoTFR_VN As New CCursorField(Of Double)("RapportoTFR_VN")
        Private m_Rating As New CCursorField(Of Integer)("Rating")
        Private m_DataRevisione As New CCursorField(Of Date)("DataRevisione")
        Private m_DataScadenzaRevisione As New CCursorField(Of Date)("DataScadenzaRevisioneione")
        Private m_StatoAzienda As New CCursorFieldObj(Of String)("StatoAzienda")
        Private m_DettaglioStatoAzienda As New CCursorFieldObj(Of String)("DettaglioStatoAzienda")
        Private m_GiorniAnticipoEstinzione As New CCursorField(Of Integer)("GiorniAnticipoEstinzione")
        Private m_Flags As New CCursorField(Of CQSPDValutazioneAziendaFlags)("Flags")

        Public Sub New()
        End Sub

        Public ReadOnly Property IDOperatore As CCursorField(Of Integer)
            Get
                Return Me.m_IDOperatore
            End Get
        End Property

        Public ReadOnly Property NomeOperatore As CCursorFieldObj(Of String)
            Get
                Return Me.m_NomeOperatore
            End Get
        End Property

        Public ReadOnly Property IDAzienda As CCursorField(Of Integer)
            Get
                Return Me.m_IDAzienda
            End Get
        End Property

        Public ReadOnly Property NomeAzienda As CCursorFieldObj(Of String)
            Get
                Return Me.m_NomeAzienda
            End Get
        End Property

        Public ReadOnly Property TipoFonte As CCursorFieldObj(Of String)
            Get
                Return Me.m_TipoFonte
            End Get
        End Property

        Public ReadOnly Property IDFonte As CCursorField(Of Integer)
            Get
                Return Me.m_IDFonte
            End Get
        End Property

        Public ReadOnly Property NomeFonte As CCursorFieldObj(Of String)
            Get
                Return Me.m_NomeFonte
            End Get
        End Property

        Public ReadOnly Property CapitaleSociale As CCursorField(Of Decimal)
            Get
                Return Me.m_CapitaleSociale
            End Get
        End Property

        Public ReadOnly Property NumeroDipendenti As CCursorField(Of Integer)
            Get
                Return Me.m_NumeroDipendenti
            End Get
        End Property

        Public ReadOnly Property FatturatoAnnuo As CCursorField(Of Decimal)
            Get
                Return Me.m_FatturatoAnnuo
            End Get
        End Property

        Public ReadOnly Property RapportoTFR_VN As CCursorField(Of Double)
            Get
                Return Me.m_RapportoTFR_VN
            End Get
        End Property

        Public ReadOnly Property Rating As CCursorField(Of Integer)
            Get
                Return Me.m_Rating
            End Get
        End Property

        Public ReadOnly Property DataRevisione As CCursorField(Of Date)
            Get
                Return Me.m_DataRevisione
            End Get
        End Property

        Public ReadOnly Property DataScadenzaRevisione As CCursorField(Of Date)
            Get
                Return Me.m_DataScadenzaRevisione
            End Get
        End Property

        Public ReadOnly Property StatoAzienda As CCursorFieldObj(Of String)
            Get
                Return Me.m_StatoAzienda
            End Get
        End Property

        Public ReadOnly Property DettaglioStatoAzienda As CCursorFieldObj(Of String)
            Get
                Return Me.m_DettaglioStatoAzienda
            End Get
        End Property

        Public ReadOnly Property GiorniAnticipoEstinzione As CCursorField(Of Integer)
            Get
                Return Me.m_GiorniAnticipoEstinzione
            End Get
        End Property

        Public ReadOnly Property Flags As CCursorField(Of CQSPDValutazioneAziendaFlags)
            Get
                Return Me.m_Flags
            End Get
        End Property

        Protected Overrides Function GetModule() As CModule
            Return Finanziaria.ValutazioniAzienda.Module
        End Function

        Public Overrides Function GetTableName() As String
            Return "tbl_CQSPDValutazioniAzienda"
        End Function


        Protected Overrides Function GetConnection() As CDBConnection
            Return Finanziaria.Database
        End Function

    End Class


End Class
